src/MSBuild.UnitTests/XMake_Tests.cs

```diff
@@ -1250,7 +1251,6 @@ private void RunPriorityBuildTest(ProcessPriorityClass expectedPrority, params s
             IDictionary<string, string> environmentVars = new Dictionary<string, string>
             {
                 { "MSBUILDNOINPROCNODE", "1"},
-                { "DISABLECONSOLECOLOR", "1"},
```

Comment:
Why this change?

Reply:
This was part of trying to figure out why it was hanging, but I forgot to revert it. Will fix.